### PR TITLE
leak: fix memory leak introduced in #272

### PR DESCRIPTION
**Pull request by algebrait**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added

### CODE REVIEW

**Comment by algebrait** [comment]
If open valgrind, ci time will increase.

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "lint": "tools/check_tidy.py && eslint .",
     "test": "npm run lint && npm run test-jerry && npm run test-iotjs",
     "test-jerry": "deps/jerry/tools/run-tests.py --unittests",
-    "test-iotjs": "tools/build.py --run-test --no-check-valgrind",
+    "test-iotjs": "tools/build.py --run-test",
```
Comment (yorkie) [inline package.json]: @algebrait CI doesn't use this to enable/disable valgrind, and I think we might try to open it at [this line](https://github.com/Rokid/ShadowNode/blob/master/tools/travis_script.py#L27)?

**Comment by algebrait** [comment]
travis problem is tracked by #319

**Comment by yorkie** [comment]
Thanks for this fix @algebrait :)

**Pull request merged**